Assets/Live2D/Cubism/Core/Unmanaged/CubismUnmanagedArrayView.cs

```diff
@@ -88,7 +88,7 @@ public sealed class CubismUnmanagedFloatArrayView
         /// <summary>
         /// Unmanaged buffer address.
         /// </summary>
-        private unsafe float* UnmanagedFixedAddress { get; set; }
+        internal unsafe float* UnmanagedFixedAddress { get; set; }
```

Comment:
We cannot accept this change either, as it is related to the modifications in `Assets/Live2D/Cubism/Core/ArrayExtensionMethods`.